corehq/apps/userreports/transforms/specs.py

```diff
@@ -109,8 +109,11 @@ class TranslationTransform(Transform):
     """
     type = TypeProperty('translation')
     translations = DictProperty()
+    mobile_or_web = StringProperty(default="web", choices=["mobile", "web"])
```

Comment:
also, if we are making the default "web" (which makes sense to me) don't we need a data migration since all current users of this feature are using it for mobile?